--- origsrc/ocaml-flac-0.1.0/configure	2011-07-04 15:45:44.000000000 -0500
+++ src/ocaml-flac-0.1.0/configure	2011-12-15 19:51:56.179979100 -0600
@@ -5165,7 +5165,7 @@ fi
 ac_config_headers="$ac_config_headers config.h:config.h.in"
 
 # Include it
-CFLAGS="$CFLAGS -I`pwd`/`dirname $0`"
+CFLAGS="$CFLAGS -I`pwd`"
 # Check endianess
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
--- origsrc/ocaml-flac-0.1.0/configure.ac	2011-07-04 15:45:44.000000000 -0500
+++ src/ocaml-flac-0.1.0/configure.ac	2011-12-15 19:52:00.074201800 -0600
@@ -24,7 +24,7 @@ PKG_PROG_PKG_CONFIG()
 # Include a config.h
 AC_CONFIG_HEADERS([config.h:config.h.in])
 # Include it
-CFLAGS="$CFLAGS -I`pwd`/`dirname $0`"
+CFLAGS="$CFLAGS -I`pwd`"
 # Check endianess
 AC_C_BIGENDIAN(AC_DEFINE([BIGENDIAN], [1], [The target is big endian]),[])
 
